fuel-txpool/src/txpool.rs

```diff
@@ -132,6 +131,126 @@ impl TxPool {
         }
         Ok(())
     }
+
+    /// Import Arc wrapped transaction to txpool.
```

Comment:
```suggestion
    /// Import a set of transactions external to network gossip, including from GraphQL endpoints.
```

Reply:
Comment added and i renamed `insert_inner` to just `insert` as it is the main function called for the insertion of transaction